psi4/src/read_options.cc

```diff
@@ -2775,214 +2775,238 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     if (name == "OPTKING" || options.read_globals()) {
         /*- MODULEDESCRIPTION Performs geometry optimizations and vibrational frequency analyses. -*/
 
-        /*- SUBSECTION Optimization Algorithm -*/
+        /* SUBSECTION Optimization Algorithm */
 
-        /*- Maximum number of geometry optimization steps -*/
-        options.add_int("GEOM_MAXITER", 50);
-        /*- Print all optking parameters. -*/
-        options.add_bool("PRINT_OPT_PARAMS", false);
-        /*- Specifies minimum search, transition-state search, or IRC following -*/
-        options.add_str("OPT_TYPE", "MIN", "MIN TS IRC");
-        /*- Geometry optimization step type, either Newton-Raphson or Rational Function Optimization -*/
-        options.add_str("STEP_TYPE", "RFO", "RFO RS_I_RFO P_RFO NR SD LINESEARCH");
-        /*- Geometry optimization coordinates to use.
-            REDUNDANT and INTERNAL are synonyms and the default.
-            CARTESIAN uses only cartesian coordinates.
-            BOTH uses both redundant and cartesian coordinates.
-            CUSTOM is not fully implemented yet - expected optking 0.3.1  -*/
-        options.add_str("OPT_COORDINATES", "INTERNAL", "REDUNDANT INTERNAL CARTESIAN BOTH CUSTOM");
-        /*- A string formatted as a dicitonary containing a set of coordinates. Coordinates can be
-            appended to Optking's coordinate set or used on their own - expected optking 0.3.1. -*/
+        /* Alphabetized so I can check for missing keywords more easily against sorted model */
+
+        /* */
+        options.add_bool("ACCEPT_SYMMETRY_BREAKING", false);
+        /* Do add bond coordinates at nearby atoms for non-bonded systems? */
+        options.add_bool("ADD_AUXILIARY_BONDS", true);
+        /* Skipping ALG_GEOM_MAXITER */
+        /* This factor times standard covalent distance is used to add extra stretch coordinates. */
+        options.add_double("AUXILIARY_BOND_FACTOR", 2.5);
+        /* SKIPPING BT_X KEYWORDS */
+        /* Do read Cartesian Hessian?  Only for experts - use
+        :py:attr:`~optking.v1.optparams.OptParams.full_hess_every` instead. */
+        options.add_bool("CART_HESS_READ", false);
+        /* */
+        options.add_str("CONJUGATE_GRADIENT_TYPE", "");
```

Comment:
```suggestion
        /*- :py:attr:`~optking.v1.optparams.OptParams.conjugate_gradient_type` -*/
        options.add_str("CONJUGATE_GRADIENT_TYPE", "");
```
not mandatory at all. I _think_ the above suggestion would be a way to get the normal autodoc file generated for the option w/o much code duplication. I don't have a strong opinion at the moment on whether that's worth it.

Reply:
I'm not opposed if having optking's options in the main keyword list feels important to anyone - that seems fairly easy to maintain. Currently, there are three copies of optking's options `v1/optparams.py`, `v2/optparams.py`, and `psi4/read_options.cc` so I just wanted to avoid having to copy documentation over again.

Reply:
Added links.